### ISSUE TITLE
Tiny GUI text on openSUSE Tumbleweed

### ISSUE THREAD

**Issue by kreaninw** [cdgriffith/FastFlix#367]
I launch **[FastFlix_4.10.0_ubuntu_22_x86_64.zip](https://github.com/cdgriffith/FastFlix/releases/download/4.10.0/FastFlix_4.10.0_ubuntu_22_x86_64.zip)** with: `QT_QPA_PLATFORM=wayland ./FastFlix`. 

The app opens but hardly usable due to tiny text, as shown in the screenshot below:

![Screenshot from 2022-08-09 04-36-29](https://user-images.githubusercontent.com/1403194/183606120-76a8b75e-d9ac-459c-9b80-01364013b4b9.png)

The terminal shows:
```
$ QT_QPA_PLATFORM=wayland ./FastFlix
2022-08-09 04:47:40 localhost.localdomain fastflix-core[29017] INFO Starting FastFlix 4.10.0
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
2022-08-09 04:47:41 localhost.localdomain fastflix[29026] DEBUG Using config file /home/archerallstars/.local/share/FastFlix/fastflix.yaml
2022-08-09 04:47:41 localhost.localdomain fastflix[29026] INFO Running command: /usr/bin/ffmpeg -version
2022-08-09 04:47:41 localhost.localdomain fastflix[29026] INFO Running command: /usr/bin/ffprobe -version
2022-08-09 04:47:41 localhost.localdomain fastflix[29026] INFO Running command: /usr/bin/ffmpeg -hide_banner -encoders
2022-08-09 04:47:41 localhost.localdomain fastflix[29026] INFO Running command: /usr/bin/ffmpeg -hide_banner -log_level error -init_hw_device opencl -h
2022-08-09 04:47:42 localhost.localdomain fastflix[29026] DEBUG Checking for newer versions of FastFlix
2022-08-09 04:47:44 localhost.localdomain fastflix[29026] DEBUG FastFlix is up tp date
2022-08-09 04:48:06 localhost.localdomain fastflix-core[29017] DEBUG Conversion worker shutting down
```

**Comment by cdgriffith** [comment]
Hi @kreaninw does [5.1.0](https://github.com/cdgriffith/FastFlix/releases/tag/5.1.0) fix this issue?

**Comment by kreaninw** [comment]
Hi @cdgriffith **[5.1.0](https://github.com/cdgriffith/FastFlix/releases/tag/5.1.0)** doesn't fix this issue, as shown in the screenshot below:

![Screenshot from 2022-12-03 13-41-17](https://user-images.githubusercontent.com/1403194/205451795-d39dcb06-8e84-401e-a203-fb2f572a83d6.png)

Here is my terminal return:

```
./FastFlix
2022-12-03 13:40:42 localhost.localdomain fastflix-core[16881] INFO Starting FastFlix 5.1.0
Gtk-Message: 13:40:42.282: Failed to load module "lunar-calendar-module"
2022-12-03 13:40:42 localhost.localdomain fastflix[16887] DEBUG Using config file /home/archerallstars/.local/share/FastFlix/fastflix.yaml
2022-12-03 13:40:42 localhost.localdomain fastflix[16887] INFO Running command: /usr/bin/ffmpeg -version
2022-12-03 13:40:43 localhost.localdomain fastflix[16887] INFO Running command: /usr/bin/ffprobe -version
2022-12-03 13:40:43 localhost.localdomain fastflix[16887] INFO Running command: /usr/bin/ffmpeg -hide_banner -encoders
2022-12-03 13:40:43 localhost.localdomain fastflix[16887] INFO Running command: /usr/bin/ffmpeg -hide_banner -log_level error -init_hw_device opencl -h
2022-12-03 13:40:44 localhost.localdomain fastflix[16887] DEBUG Checking for newer versions of FastFlix
2022-12-03 13:40:46 localhost.localdomain fastflix[16887] DEBUG FastFlix is up tp date
2022-12-03 13:41:28 localhost.localdomain fastflix-core[16881] DEBUG Conversion worker shutting down
```

**Comment by cdgriffith** [comment]
I am changing how it looks for fonts in next release, but it also seems that by default `openSUSE` doesn't come out of the box with any fonts PySide6 can use. When I first ran it I actually only got boxes for letters, so I imagine it's just defaulting to whatever font it can find to use. 


After installing `ubuntu-fonts` and `texlive-helvetic-fonts` it looks more natural, and hopefully better when 5.2.0 comes out. (This is all through WSL so may be different than actual system)

**Comment by cdgriffith** [comment]
Hopefully fixed in [5.2.0](https://github.com/cdgriffith/FastFlix/releases/tag/5.2.0)! (please re-open if not)

**Comment by kreaninw** [comment]
I confirm this is fixed, tested in **[5.2.2](https://github.com/cdgriffith/FastFlix/releases/tag/5.2.2)**, as shown in the screenshot below:

![Screenshot from 2023-03-10 21-35-21](https://user-images.githubusercontent.com/1403194/224453864-c687b845-d71c-4856-8be0-56cf35df3c02.png)
